src/Components/StarPrompt/index.jsx

```
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react';
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)